### DISCUSSION TITLE
How useful is the DE (Dry End) calculator & what is it based on?

### DISCUSSION THREAD

**Discussion by Javapik** [artisan-roaster-scope/artisan#731] [Q&A] [answered]
Greets. Just got the word about the demise of the mailing list and had been meaning to ask this question. The running time calculator at the top of the screen that estimates when 'Dry End' will occur, what is it based on? It looks like it just marks when the Bean temperature hits 300f. The outcome is affected by changing the heat and/or fan on my Hottop and I've been trying to use it to stretch that phase of the roast. I realized that Bean color and smell should be my guide but I've been wondering how useful of a tool this is.

**Comment by MAKOMO** [comment]
With "Auto Adjusted" phases the DRY event is set from the temperature limit defined under menu Config >> Phases. Without, you need to use the DRY key to set it where it occurs. The running time calculator is based on the time set for the DRY event, however it was set.

**Comment by poundy** [comment, marked as answer]
To answer the "what is it based on" question, just confirming you're talking about using the "projected" dry-end-point to stretch or collapse the time where that is achieved, all before the event has occurred.  That is based on the temperature limit that is in Config --> Phases, as Marko has said above, taking into account the current conditions. If your weather/bean/roaster combination you always see dry-end at a higher or lower temperature, then there's no harm in changing that to what you think the dry-end typically is and the calculation will still use that to predict the time it will reach DE. 

"Is it useful"  is a bit harder to answer, but I would say absolutely - it's a critical milestone in coffee roasting, but whether modulating RoR the way you're doing it is a good practice, well thats what cupping is for. For me, I have an aim DE point for a coffee, and use timing of heat reapplication after charge and early roast stages to establish that momentum to achieve it at or around my aim time.  I don't change airflow dramatically on my roaster, but it's not a hottop either!  When you say "stretch", how long are you talking about the change? from 4:50 to 5:10? or 4:50 to 6:00? Those can and will have wildly different impacts on the coffee...

**Comment by Javapik** [comment]
Thanks, that helps my understanding, I will have to review my Config---Phases. I don't remember ever setting it. Poundy, my attempts have been more like 4:55 to 5:10. Any longer and it really slows my momentum with the Brazil,Ethiopia, Costa Rican blend I usually roast for espresso.
